api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -274,10 +285,39 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	 */
 	public List<Concept> getConceptsWithDrugsInFormulary() throws DAOException;
 	
+	/**
+	 * Creates a new Concept name tag if none exists. If a tag exists with the same name then that
+	 * existing tag is returned.
+	 * 
+	 * @param nameTag the concept name tag to be saved
+	 * @return the newly created or existing concept name tag
+	 * @should save a concept name tag if tag does not exist
+	 * @should not save a concept name tag if tag exists
+	 * @should not save a concept name tag if tag is null, empty or whitespace
+	 * @should save a concept name tag if tag is supplied
+	 * @should save an edited concept name tag
+	 */
```

Comment:
Since this method is well documented in the `ConceptService` interface, don't you wanna reference to that documentation with the help of `@see` javadoc attribute just like its done in other places in this class?